### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by sophieheb**

### CODE REVIEW

File: app/views/layouts/errors.html.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <link href="https://fonts.googleapis.com/css?family=Nunito:300,300i,400,400i,700,700i&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
```
Comment (jits) [inline app/views/layouts/errors.html.erb]: If the head content is the same as the main layout, could DRY up in a `_head` partial.

Comment (sophieheb) [reply app/views/layouts/errors.html.erb]: Done!

File: app/views/errors/forbidden.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<h2 class="text-center">Sorry! You don’t have a permission to access this area.</h2>
+<div class='row'>
+  <img src="https://a.storyblok.com/f/50602/479x450/7938231392/not-the-best-looking-cat.png" class="img-fluid mx-auto" alt="...">
+</div>
+
+<!-- main content -->
+<div class="container-fluid col-text">
+  <div class="py-3 px-lg-5 col-12">
+    <p class="lead text-center">
+      Head back to the <a href="https://soulmedicine.io" target="_blank">Homepage</a> or if you feel this is an error let us know at <i>team at soulmedicine.io</i>
```
Comment (jits) [inline app/views/errors/forbidden.html.erb]: Would it make more sense not to open this link in a new tab?

Comment (jits) [reply app/views/errors/forbidden.html.erb]: (and elsewhere)

**Pull request merged**